client/src/components/MultiJurisdictionAudit/_mocks.ts

```diff
@@ -445,6 +451,12 @@ export const aaApiCalls = {
     url: '/api/election/1/standardized-contests/file',
     response,
   }),
+  getMapData: {
+    url: 'https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json',
+    response: new Response(
+      'https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json'
+    ),
```

Comment:
does this mean our tests rely on connecting to this CDN now? wondering if we can just put in some fake local data instead